configure.py

```diff
@@ -1042,8 +1049,9 @@ def get_gas_version(cc):
 # quite prepared to go that far yet.
 def check_compiler(o):
   if sys.platform == 'win32':
-    o['variables']['clang'] = 0
```

Comment:
I made a mistake here, sorry. I'll keep the lines setting to 0 when the flag isn't passed

Reply:
Fixed